tslint.json

```diff
@@ -14,7 +14,16 @@
 			2
 		],
 		"no-conditional-assignment": false,
-		"arrow-parens": false
+		"arrow-parens": false,
+		"no-redundant-jsdoc": true,
+		"completed-docs": [
+		  true,
+		  {
+		    "functions": {"visibilities": ["exported"]},
+		    "methods": {"privacies": ["public", "protected"]}
+		  }
+		],

```

Comment:
Forces to have JSDocs for all exported functions as well as public and protected methods.